### ISSUE TITLE
Allow bmcsetup to be run outside of genesis

### ISSUE THREAD

**Issue by mattaezell** [xcat2/xcat-core#2343]
There are a couple reasons why you might want to run bmcsetup while nodes are online (in an osimage, not genesis)
- You forgot to put _runcmd=bmcsetup_ in the chain table
- You want to change the BMC username/password

I was able to copy the relevant scripts to the node and modify them to run outside of genesis
- Change the logging to remove genesis (cosmetic only)
- Remove /tmp/ipmicfg.xml at the end (optional, but good for security)
- Change getipmi to pull XCAT=<server:port> from /proc/cmdline

```
# updatenode node1 -P bmcsetup
node1: xcatdsklspost: downloaded postscripts successfully
node1: Wed Dec 21 10:52:50 EST 2016 Running postscript: bmcsetup
node1: postscript: bmcsetup exited with code 0
node1: Running of postscripts has completed.
```

Ideally, the scripts would only live in the xCAT repo in one place so that updates would happen inside genesis and /install/postscripts

**Comment by immarvin** [comment]
hi @zet809 would you please take a look at this?

**Comment by zet809** [comment]
@mattaezell I have modified the 2 scripts: bmcsetup and getipmi to work both within genesis and as postscript for diskful/diskless OS provisioning. They are available at #2347
We need to discuss whether include them in xCAT-server package directly.

**Comment by zet809** [comment]
Those 2 script had been included in xcat package, so close this defect.
